go.mod

```diff
@@ -87,6 +87,7 @@ require (
 )
 
 replace (
+	github.com/apache/yunikorn-release => github.com/apache/yunikorn-release v1.1.0-1.0.20250213044724-3a0499434067
```

Comment:
Could you use a proper master version?

Reply:
So far we are looking to merge all PRs into a branch and then later commit those into master branch. Added this to point to a particular branch (soak-test)